xayn-ai/src/ranker/context.rs

```diff
@@ -13,10 +13,10 @@ pub(crate) struct Context {
 impl Context {
     pub(crate) fn from_cois(cois: &[(Id, CoiComponent)]) -> Self {
         let cois_len = cois.len() as f32;
-        let pos_avg = cois.iter().map(|(_, coi)| coi.pos_distance).sum::<f32>() / cois_len;
+        let pos_avg = cois.iter().map(|(_, coi)| coi.pos_similarity).sum::<f32>() / cois_len;
```

Comment:
the `calculate_score()` below was meant for distances, i guess it has to be updated to use similarities as well now? if the formulas stay the same, we probably still have to update the default values in the unwrappings.